wire-schema/src/jvmMain/java/com/squareup/wire/schema/MarkSet.kt

```diff
@@ -34,6 +34,10 @@ class MarkSet(
 
   /** The members to retain. Any member not in here should be pruned! */
   val members = mutableMapOf<ProtoType, MutableSet<ProtoMember>>()
+  /** The root members which are never to be pruned, including their actual type. */
```

Comment:
“actual type“ means referenced type? I want a better word but I can’t think of one

Reply:
I prefer _referenced type_ here, which is what you use on line 66 anyway

Reply:
fixed